symfit/core/argument.py

```diff
@@ -115,6 +115,23 @@ def __init__(self, name=None, value=1.0, min=None, max=None, fixed=False, **assu
             self.min = min
             self.max = max
 
+    def __eq__(self, other):
+        """
+        Parameters are considered equal when their name, assumptions, and
+        bounds are considered the same.
+        """
+        equal = super(Parameter, self).__eq__(other)
+        if equal is NotImplemented:
+            return equal
+        else:
```

Comment:
Pointless else

Reply:
I would still invert this condition: 
```python3
if not equal:
    return False
return (self.min == other.min and
        self.max == other.max and
        self.fixed == other.fixed and
        self.value == other.value)
```

Reply:
Still a pointless else :D

Reply:
I'm personally more of a fan of having the else there (as you probably guessed :P) because I think it improves readability.

Reply:
In this case it's not too important. But keeping the indentation under control starts to matter for larger blocks of code and nested constructs